src/main/webapp/clinics-map.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Planned Parenthood Clinic Locations</title>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyB1B3SxGUn4uJexYA-8wb5IWSxkPlZyKe8"></script>
+  <script>  
+    function createClinicsMap(){
+      fetch('/clinics-data').then(function(response) {
+        return response.json();
+      }).then((Clinic) => {
```

Comment:
Use 'clinic' instead of 'Clinic'. Also shouldn't it be clinics, since there are multiple ones

Reply:
Changed to clinics